AsyncCodecSource.cpp

```diff
@@ -285,19 +284,16 @@ status_t AsyncCodecSource::stop() {
     return res1 != OK ? res1 : res2;
 }
 
-sp<MetaData> AsyncCodecSource::getFormat() {
+sp<MetaData> AsyncCodecSource::getFormat()
+{
     Mutexed<sp<MetaData>>::Locked meta(mMeta);
     return *meta;
 }
 
-AsyncCodecSource::Output::Output()
-    : mReachedEOS(false),
-      mReading(false) {
-}
+AsyncCodecSource::Output::Output() : mReachedEOS(false), mReading(false) { }
```

Comment:
I'd prefer here the init list separate and wouldn't like folding the {} body to the line.